webapp-django/crashstats/base/views.py

```diff
@@ -3,6 +3,16 @@
 from django.shortcuts import render
 
 
+class HttpResponseBadRequest(http.HttpResponseBadRequest):
```

Comment:
This is a Socorro-enhanced version, so it should have a different name to make it easier to know in the code when you're using the right one especially where this has security-related consequences.